lib/matplotlib/figure.py

```diff
@@ -56,7 +56,7 @@ class _AxesStack(cbook.Stack):
     Specialization of `.Stack`, to handle all tracking of `~.axes.Axes` in a
     `.Figure`.
 
-    This stack stores ``key, (ind, Axes)`` pairs, where:
+    This stack stores ``key, (ind, axes)`` pairs, where:
```

Comment:
Usage in literals is similar to variables and thus should stay lowercase.

Reply:
Whoops I forgot to mention this in the comments. I wasn't sure about this one. Thanks for fixing.

Reply:
I thought it was fine, but I'm so old I remember when everyone was excited about mixed-case variable names (an improvement from all-caps!)